doc/interop-test-descriptions.md

```diff
@@ -130,11 +138,39 @@ Procedure:
       zero and comparing the entire response message against a golden response
 
 
- 2. Client calls UnaryCall with:
+### server_compressed_unary
+
+This test verifies the server can compress unary messages. It sends two unary
+requests for a COMPRESSABLE payload type, expecting the server response to be
```

Comment:
Ah, I see what the "requests for a compressable payload" means now. It means I send a request with "response_type" COMPRESSABLE, which means  that I'm requesting responses with payload type COMPRESSABLE.
I think it's confusing.

How about:

Clients sends two unary requests. The first one instructs the server to send a compressed response (COMPRESSABLE payload type). The other one instructs the server to send an uncompressed response (again of COMPRESSABLE payload). 

Client then asserts if compression was actually performed is by inspecting the compression bit in the response's message flags (currently only exposed in C/C++ code).